ansible/roles/cron/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: Create crontabs log directory
+  file: path=/var/log/crontabs state=directory owner={{http_user}} group={{http_user}}
+- name: Copy run_script crontabs
```

Comment:
No reason, except that I didn't know about that module ^^ It would have probably been easier using it!